### PR TITLE
Create default form section [invisible]

### PR DESCRIPTION
**Pull request by llvasconcellos**
This fixes #204 -> "Failed to fetch form from server" problem.

### CODE REVIEW

File: tools/profile_apply [hunk 1]
```diff
@@ -413,6 +413,7 @@ def apply(tabs):
         field_id = get_field_for_element('encounter', 'provider_id')
         add_field_to_form(form_id, field_id, section_id, sort_weight=3)
 
+        begining = True
```
Comment (capnfabs) [inline tools/profile_apply]: `begining` should be `beginning` :)

File: tools/profile_apply [hunk 2]
```diff
@@ -440,13 +442,18 @@ def apply(tabs):
...
                     section_id = add_field_to_form(form_id, field_id, None, fn)
-                elif last_type == 'select_multiple':
+                elif last_type == 'select_multiple' or begining:
                     # Fields after the end of a toggle group don't belong in
                     # the section, so put them in a new section, using
                     # "[invisible]" in its name to hide the section heading.
+                    #
+                    # If begining is True then there is no section declared in
+                    # the begining of the form so we also need to create at least
+                    # one "[invisible]" section to avoid chart rendering problems.
```
Comment (capnfabs) [inline tools/profile_apply]: Can you change this to `one "[invisible]" section to work around a bug in the ODK Android library.` ?

**Comment by capnfabs** [comment]
Looks good, ok to merge.

File: tools/profile_apply (hunk 2 above)
Comment (llvasconcellos) [inline tools/profile_apply]: I meant form rendering not chart! Where was my head at?

File: tools/profile_apply (hunk 1 above)
Comment (llvasconcellos) [inline tools/profile_apply]: Well, I could have written "começo" (beginning in portuguese). :)

**Pull request merged**